client/src/app/Log.less

```diff
@@ -1,12 +1,16 @@
-:local(.Log) {
+:root {
```

Comment:
Any reason to not put these vars inside the `local`, seeing as they are to be used only within the Log scope? Is it just good practise?

Reply:
The reason is that we want to be able to easily override the styling.

For that to be possible, we must declare variables on `:root`.

Reply:
Ah, right. Makes sense